access/services/project.go

```diff
@@ -0,0 +1,160 @@
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/jfrog/jfrog-client-go/auth"
+	"github.com/jfrog/jfrog-client-go/http/jfroghttpclient"
+	clientutils "github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/httputils"
+)
+
+const projectsApi = "v1/projects"
```

Comment:
I think it would be clearer if we add the `api` to the const, as we do in some of Artifactory's services:
```suggestion
const projectsApi = "api/v1/projects"
```